app/Jobs/NotifyOutdatedTraefikServersJob.php

```
@@ -0,0 +1,63 @@
+
+            return true;
+        });
+
+        if ($outdatedServers->isEmpty()) {
+            return;
+        }
+
+        $outdatedServers
+            ->groupBy('team_id')
+            ->each(function ($teamServers, int|string $teamId): void {
+                $team = Team::find($teamId);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Models\Team' in method 'handle'. (undefined)

(StaticAccess)